node/src/components/chain_synchronizer/operations.rs

```diff
@@ -592,9 +597,11 @@ async fn fast_sync_to_most_recent(
                     .take_header();
         }
     }
+    info!("finish - fetch and store all blocks - fast sync");
```

Comment:
I'd keep them for now as I'd really like to avoid that we miss some important debug trace in the overnight test.

After all, they all should be removed, being replaced by proper measurement tools.

Let me just change the "finish" messages to `debug`-level.